doc/examples/basic.js

```diff
@@ -0,0 +1,30 @@
+var seneca = require('seneca')()
+seneca.use(require('seneca-basic'))
+seneca.use(require('seneca-entity'))
+seneca.use(require('seneca-user'))
+
+seneca.ready((msg, done) => {
+  seneca.act({role: 'user', cmd: 'register', name: "Flann O'Brien", email: 'nincompoop@deselby.com', password: 'blackair'},
```

Comment:
@mihaidma Can you run through a sample of using jsonic strings with payloads with @eeswr ? We should have user/cmd as a jsonic string, the rest as payload